salt/beacons/status.py

```diff
@@ -116,9 +117,11 @@ def beacon(config):
             'time': 'all',
         }
 
-    ret = {}
     for func in config:
-        data = __salt__['status.{0}'.format(func)]()
+        try:
+            data = __salt__['status.{0}'.format(func)]()
+        except (KeyError, AttributeError):
+            continue
```

Comment:
It is merely reflecting the changes that are present for beacon/status.py in formulas.
Did not want to diverge from that.

Real issue is that AIX python does not support os.getloadavg, which it really should and just do what they did so ofter in XOpen/98, return 'Not Implemented', that or Zero.

Secondly this beacon status is being used to determine presence, even if it returns not available for cpuinfo, meminfo, etc.